metadata-service/openapi-servlet/src/main/java/io/datahubproject/openapi/v3/OpenAPIV3Generator.java

```diff
@@ -434,6 +437,10 @@ private static void addAspectSchemas(final Components components, final AspectSp
                   final String newDefinition =
                       definition.replaceAll("definitions", "components/schemas");
                   Schema s = Json.mapper().readValue(newDefinition, Schema.class);
+                  // Set nullable attribute
+                  Optional.ofNullable(s.getProperties()).orElse(new HashMap())
+                          .forEach((name, schema) -> ((Schema) schema).setNullable(!Optional.ofNullable(s.getRequired())
+                                  .orElse(new ArrayList()).contains(name)));
```

Comment:
Set nullable for all aspect attributes that are not required.
During the conversion from PDL -> Avro -> jsonschema, `optional` fields will not be marked as `required` correctly but they are missing the `nullable`